admin_ui/src/components/EditRowForm.vue

```diff
@@ -110,7 +112,13 @@ export default Vue.extend({
                 }
                 this.$store.commit("updateApiResponseMessage", message)
             } catch (error) {
-                const data = error.response.data
+                let data = error.response.data
+
+                if (data["db_error"]) {
+                    data = data["db_error"]
+                } else {
+                    data = data["detail"][0]["msg"]
+                }
```

Comment:
Lower down in the file we have `if (typeof data != "string") {`

We should try and merge both pieces of logic.

Reply:
<img width="384" alt="Screenshot 2022-09-07 at 11 16 59" src="https://user-images.githubusercontent.com/350976/188854280-6a80d183-9c87-4091-88d9-3b5c58befecb.png">

If the error message is from Pydantic, we also need to use the `loc` value so we can tell the user which field has the problem.

Reply:
I'd consider adding a utility function like `parsePydanticError` or something, so we can re-use the logic for converting the error into a nice string.

Reply:
We might also be able to distinguish between Pydantic, database errors, and other errors based on the status code.

Reply:
As I understand it, we have two types of messages (database error from Piccolo API `db_exception_handler` with status code 422 and Pydantic validition errors). I can add `loc` to the Pydantic error message